README.md

```diff
@@ -1,13 +1,20 @@
 # BorgBackup [Homebrew](https://brew.sh/) tap
 
-## How do I install formulas?
+## Why install [Borg](https://www.borgbackup.org/) using this tap vs `brew install borgbackup`?
+
+The `borgbackup-llfuse` formula maintained in this tap depends on [osxfuse](https://formulae.brew.sh/cask/osxfuse) and [llfuse](https://github.com/python-llfuse/python-llfuse/) which are required to mount repositories or archives using `borg mount`.
+
+These dependencies have been [removed](https://github.com/Homebrew/homebrew-core/commit/8c2f17e3b653347ada86d353243e2d6b6cb10fda#diff-4a25217474a5eb61d0776ab4cabc43b42689bc7b3efaaed400f799631dcec71f) from Homebrew’s [borgbackup](https://formulae.brew.sh/formula/borgbackup) formula because [FUSE for macOS](https://osxfuse.github.io/) is no longer open source.
```

Comment:
here it talks about fuse for macos, above it is named osxfuse - migh be confusing for people who do not know it is referring to the same project.

Reply:
@ThomasWaldmann I agree the naming convention of Homebrew vs the repo vs the website is confusing... to make things worse, FUSE for macOS has been renamed to macFUSE starting at version 4. What do you suggest? I guess we could use `osxfuse` as this is the name of the formula?

Reply:
it could at both places call it osxfuse.

at first place it could also say `(aka FUSE for macOS)`.